### ISSUE TITLE
When running "CMake build" and "CMake clean", the quickfix window closes automatically.

### ISSUE THREAD

**Issue by csy19960309** [Shatur/neovim-cmake#10]
**Bug description**
Just like what is described in the title.

**Steps to reproduce**

<details>
<div>
   init.lua:
</div>
<div> 
require('cmake').setup({
     parameters_file = 'neovim.json', 
     build_dir = '{cwd}/build/{os}-{build_type}', 
     default_projects_path = '~/Projects', 
     configure_arguments = '-D CMAKE_EXPORT_COMPILE_COMMANDS=1 -G Ninja',      
    build_arguments = '-j8', 
     asyncrun_options = { save = 2 }, 
     target_asyncrun_options = {}, 
     dap_configuration = { type = 'cpp', request = 'launch' }, 
     dap_open_command = require('dap').repl.open  })
</div>
<div>
neovim.json:
</div>
<div>
{"arguments": [], "currentTarget": "all", "buildType": "Debug"}
</div>

<summary>Minimal configuration</summary>
</details>

**Expected behavior**
The QuickFix window stays open after I run the commands mentioned above so that I can see the results. This feature was available  in the last few versions but seems to be broken in the latest version.

**Screenshots**
If a GIF is needed, I will try to make one.

**Environment**

- OS:  Arch Linux in WSL2.
- Desktop environment: None.
- Plugins commit hash: 32be7c16950996dfd8e2423dff2a0366a96961f2

**Additional context**
None.

**Comment by csy19960309** [comment]
> Hi! I can't reproduce the issue with your configuration on my side. After running `:CMake build` or `:CMake clean` the quickfix window stays open until I close it by myself.

Is there any way to trace this problem? I am new to neovim and do not know much about logging.

**Comment by Shatur** [comment]
Just disable other plugins and see if it works. You need to detect which one causes your issue.

**Comment by csy19960309** [comment]
> Could you check if the latest master fixes your issue?

No, it still closes.

**Comment by csy19960309** [comment]
> Could you try to comment out this line?
> 
> https://github.com/Shatur/neovim-cmake/blob/5a2db7c207988fb317abde9ab83cf17ba9606fda/lua/cmake/utils.lua#L146

Yeah. The auto-close disappeared with that line commented out. Is this auto-close a new feature or something else?

**Comment by Shatur** [comment]
Thanks! No, it was before. I used it to automatically hide quickfix if someone was using terminal output. But I decided to remove this functionality, since the user himself will be able to automate this if he needs to.
But it's weird that I can't reproduce the issue with exactly the same configuration.
But now it should work for you. Sorry for the inconvenience.
